sugaroid/brain/constants.py

```diff
@@ -527,17 +537,21 @@
     "heyyyy",
     "ahoy",
     "ello",
-    "halo",
     "hello",
+    "whats up",
+    "hey sugaroid",
+    "hey dude",
+    "hey bro",
+    "konnichiwa",
+    "ciao",
 ]
 
 HI_RESPONSES = [
     "Hi there",
     "Hola amigo!",
-    "Hello, welcome to sugaroid",
-    "Hey, how can I help you?",
-    "Hey dude, how can I help you?",
-    "Hey there",
+    "Hey, how can I be of help?",
+    "Hey bro, how can I help you?",
```

Comment:
need to use a gender neutral word instead of `bro`

Reply:
ok